tests/Examples.hs

```diff
@@ -1,7 +1,10 @@
+-- These examples are all yanked from generic-deriving,
+-- which exports equivalents as part of the package. We
+-- probably don't want to do that here, but they make for
+-- great tests!
```

Comment:
“equivalents”?

Reply:
It's just that I have honestly no idea what you mean by this word in that sentence.